contributions.json

```diff
@@ -25,11 +25,11 @@
 		},
 		"gitlens.ai.explainCommit": {
 			"label": "Explain Commit...",
-			"commandPalette": "gitlens:enabled && !gitlens:readonly && !gitlens:untrusted && gitlens:gk:organization:ai:enabled",
+			"commandPalette": "gitlens:enabled && !gitlens:readonly && !gitlens:untrusted && gitlens:gk:organization:ai:enabled && config.gitlens.ai.enabled",
```

Comment:
I think we should add a post-release follow up task to unify these into a single context key -- at least for the majority of cases where we don't care about the difference

Reply:
https://github.com/gitkraken/vscode-gitlens/issues/4283